modelerfour/modeler/modelerfour.ts

```diff
@@ -549,7 +658,8 @@ export class ModelerFour {
 
     if (!isMoreThanObject && !hasProperties) {
       // it's an empty object?
-      this.session.warning(`Schema '${name}' is an empty object without properties or modifiers.`, ['Modeler', 'EmptyObject'], aSchema);
+      // this.session.warning(`Schema '${name}' is an empty object without properties or modifiers.`, ['Modeler', 'EmptyObject'], aSchema);
+      return this.anySchema;
```

Comment:
Returning the schema seems to be a distinct change in behavior, was it missing before or is there a reason to add it now?

Reply:
We've re-declared `{type:object}`  to mean "any"  *sigh*